JSDemos/Demos/DataGrid/StateManagement/React/App.js

```diff
@@ -0,0 +1,182 @@
+﻿import React from 'react';
+import DataGrid, { Column, Editing } from 'devextreme-react/data-grid';
+import applyChanges from 'devextreme/data/apply_changes';
+import 'whatwg-fetch';
+
+const URL = "https://js.devexpress.com/Demos/Mvc/api/DataGridWebApi";
+
+const sendChange = (url, change) => {
+  switch (change.type) {
+    case "insert":
+      return sendRequest(url + "/InsertOrder", "POST", { values: JSON.stringify(change.data) });
+    case "update":
+      return sendRequest(url + "/UpdateOrder", "PUT", { key: change.key, values: JSON.stringify(change.data) });
+    case "remove":
+      return sendRequest(url + "/DeleteOrder", "DELETE", { key: change.key });
+  }
+};
+
+const sendRequest = (url, method, data) => {
+  method = method || 'GET';
+  data = data || {};
```

Comment:
You can use `const sendRequest = (url, method = 'GET', data = {}) => {` instead of this.